ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -598,7 +604,16 @@ open class DrawerController: UIViewController {
     private func updatePreferredContentSize(isExpanded: Bool) {
         isPreferredContentSizeBeingChangedInternally = true
         if isExpanded {
-            preferredContentSize.height = UIScreen.main.bounds.height
+            let screenBound: CGFloat = UIScreen.main.bounds.height
```

Comment:
nit: I think screenBound should be named screenHeight

Reply:
renamed this